src/cpl.rs

```diff
@@ -53,12 +56,13 @@ impl CslStringList {
         Ok(())
     }
 
-    /// Looks up the value corresponding to a key.
+    /// Looks up the value corresponding to `key`.
     ///
-    /// See `CSLFetchNameValue` for details.
+    /// See [`CSLFetchNameValue`](https://gdal.org/doxygen/cpl__string_8h.html#a4f23675f8b6f015ed23d9928048361a1)
+    /// for details.
     pub fn fetch_name_value(&self, key: &str) -> Result<Option<String>> {
         let key = CString::new(key)?;
-        let c_value = unsafe { gdal_sys::CSLFetchNameValue(self.as_ptr(), key.as_ptr()) };
+        let c_value = unsafe { CSLFetchNameValue(self.as_ptr(), key.as_ptr()) };
```

Comment:
This seems to be O(N^2). I'm not really familiar with this API, is there way to move to the next element of a `CSLStringList`?